.travis.yml

```diff
@@ -84,6 +88,7 @@ before_script:
   ## orocos_kdl
   - git clone https://github.com/orocos/orocos_kinematics_dynamics
   - cd orocos_kinematics_dynamics/orocos_kdl
+  - git checkout v1.4.0
```

Comment:
Why the version bump? or downgrade?

to quote ...
"To fix compilation, we will switch the dependencies to use the commits used in the latest successful model generation"

Reply:
https://github.com/robotology/icub-model-generator/pull/119#issuecomment-531726813

I will update the travis script with a general comment about how the versions are choosen.